apps/frontend/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM node:16-alpine AS base
+
+FROM base AS builder
+RUN apk add --no-cache libc6-compat
+RUN apk update
+# Set working directory
+WORKDIR /app
+RUN yarn global add turbo
+COPY . .
+RUN turbo prune --scope=@dundring/frontend --docker
+
+# Add lockfile and package.json's of isolated subworkspace
+FROM base AS installer
+RUN apk add --no-cache libc6-compat
+RUN apk add --no-cache openssl1.1-compat
+RUN apk update
+WORKDIR /app
+
+# First install dependencies (as they change less often)
+COPY .gitignore .gitignore
```

Comment:
i get an error from this line.
is this needed?

Reply:
Good questions – I just did the same as in the [Turbo docs](https://turbo.build/repo/docs/handbook/deploying-with-docker) 😅